### PR TITLE
WAI-1051: Fail when some part of the build errors

### PR DESCRIPTION
**Pull request by edmofro**
### Issue #:
WAI-1051

### CODE REVIEW

File: packages/devops/scripts/ci/triggerRedeploy.sh [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/bash -e
```
Comment (edmofro) [inline packages/devops/scripts/ci/triggerRedeploy.sh:1]: Fail the whole redeploy if any of the scripts within exit with a non-zero code

File: packages/devops/scripts/lambda/resources/startup.sh [hunk 2]
```diff
@@ -1,7 +1,9 @@
-#!/bin/bash -le
+#!/bin/bash -leE
```
Comment (edmofro) [inline packages/devops/scripts/lambda/resources/startup.sh:1]: The capital E means errors within called scripts bubble up to be caught by the top level `trap` (see below)

File: packages/devops/scripts/lambda/resources/startup.sh [hunk 3]
```diff
@@ -19,9 +28,10 @@ echo "Starting up ${DEPLOYMENT_NAME} (${BRANCH})"
 mkdir -p $LOGS_DIR
 
 # Turn on cloudwatch agent for prod and dev (can be turned on manually if needed on feature instances)
-if [[ $DEPLOYMENT_NAME == "production" || $DEPLOYMENT_NAME == "dev" ]]; then
-    $DEPLOYMENT_SCRIPTS/startCloudwatchAgent.sh | while IFS= read -r line; do printf '\%s \%s\n' "$(date)" "$line"; done  >> $LOGS_DIR/deployment_log.txt
-fi
+# TODO currently broken
```
Comment (edmofro) [inline packages/devops/scripts/lambda/resources/startup.sh:31]: This was already broken, but now that we're using pipefail on this script, it matters. We haven't really got a good solution for pushing telemetry to cloudwatch now that our instances are short-lived.

**Review by IgorNadj — APPROVED** [review]
LGTM!

**Pull request merged**